internal/skia/skia_other.go

```diff
@@ -1255,15 +1255,20 @@ func TextBlobBuilderMake(builder TextBlobBuilder) TextBlob {
 	return C.sk_textblob_builder_make(builder)
 }
 
+const sizeofUint16 = unsafe.Sizeof(uint16(0))
+const sizeofFloat32 = unsafe.Sizeof(float32(0))
+const maxUint16Array = (1<<31 - sizeofUint16) / sizeofUint16
+const maxFloat32Array = (1<<31 - sizeofFloat32) / sizeofFloat32
```

Comment:
These 4 constant declarations and the ones in the skia_windows.go file should be moved into skia.go to avoid unneeded duplication.